src/com/facebook/buck/testrunner/TestNGRunner.java

```diff
@@ -92,7 +92,6 @@ public void run() throws Throwable {
   private XmlSuite createXmlSuite(Class<?> c) {
     XmlSuite xmlSuite = new XmlSuite();
     xmlSuite.setName("TmpSuite");
-    xmlSuite.setTimeOut(String.valueOf(defaultTestTimeoutMillis));
```

Comment:
while I agree using timeout on a suite is not idea, but we should always have timeouts for tests and as such please either add timeouts for test runs or better yet consider using DelegateRunnerWithTimeout

Reply:
I thought there is always a time limit per test target specified in `.buckconfig`?

Reply:
basically, I'm afraid of introducing "non-transparent" timeouts inside Buck; users should always know that their timeout is specified explicitly somewhere, imo

Reply:
hm, well, I'm fine with any timeouts as long as there are some :) If you're saying that test execution time is already limited - I'm fine with this then :)